### PR TITLE
Initial script for Deauth attack

### PR DESCRIPTION
**Pull request by aravindraja93**
Initial script for the Deauth attack

Tobe Done:

*Refactor
*Add result checking for init_ and test_ sections

### CODE REVIEW

File: common/tests/security/deauth_attack.sh [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+_init() {
+
+    # The functions verifies if the device is an ALFA capable of monitor mode. 
+	# Upon verfication it then puts the device into monitor mode to carry out atttacks	
+  
+    echo "Init called"
+	
+	# Check if it is an ALFA device
+	addr=$(iw $interface info | grep addr | awk '{print $2}')
+	if [ ${add:0:8}=="00:c0:ca" ]; then
```
Comment (govindsi) [inline common/tests/security/deauth_attack.sh:15]: Add comment for OUI check for refrence

File: common/tests/security/deauth_attack.sh [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+}
+
+main(){
+
+
+_init
+
+_test
+
```
Comment (govindsi) [inline common/tests/security/deauth_attack.sh:135]: Define result method and call deinit

**Review by govindsi — COMMENTED** [review]
Raise change on topic branch forked from develop

File: common/tests/security/deauth_attack.sh [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+		exit 1
+	fi
+		
+}
+
+_init_result() {
+
+	# Checks if the interface is in Monitor mode or now
+
+mode=$(iwconfig wlan0 | grep Mode | awk '{print $1}' | cut -d ":" -f2-)
```
Comment (martin-tii) [inline common/tests/security/deauth_attack.sh:50]: Shouldn't we add interface as input parameter?? or is it $interface??

**Pull request closed without merging**